index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="main">
```

Comment:
I kindly suggest the use of semantic tags, like menu and header tags. Because a semantic tag clearly describes its meaning to both the browser and the developer.